### DISCUSSION TITLE
Can it run on p 100

### DISCUSSION THREAD

**Discussion by sb1992** [SYSTRAN/faster-whisper#643] [Q&A]
Looks like is pretty hard to get hold of gpus at the moment. Was wondering if can run it on p 100? I tried a bit and always get the below error
"CUDA failed with error CUDA driver version is insufficient for CUDA runtime version"
so wasn't sure if

**Comment by trungkienbkhn** [comment]
Hello. Have you installed cuBLAS & cuDNN ?
https://github.com/SYSTRAN/faster-whisper?tab=readme-ov-file#gpu
